models/DataObject/SelectOptions/Config.php

```diff
@@ -104,6 +108,21 @@ public function setGroup(?string $group): static
         return $this;
     }
 
+    public function getAdminOnly(): bool
+    {
+        return $this->adminOnly ?? false;
```

Comment:
The `$adminOnly` property is not nullable, so this is not necessary:

```suggestion
        return $this->adminOnly;
```

Reply:
Good point. It was nullable initially, but I forgot to update this.